src/yadism/coefficient_functions/light/kernels.py

```diff
@@ -197,15 +193,14 @@ def nc_fl11_weights(coupling_constants, Q2, nf, skip_heavylight=False):
         w = coupling_constants.get_fl11_weight(
             q, Q2, nf, "VV"
         ) + coupling_constants.get_fl11_weight(q, Q2, nf, "AA")
-        ns_partons[q] = w
-        ns_partons[-q] = w
+        quark_partons[q] = w
+        quark_partons[-q] = w
         tot_ch_sq += w
 
-    # compute gluon, singlet (S) (omitting the *2/2)
-    # NOTE: what we call singlet (S) is actually the pure singlet (PS) coefficient.
-    # Here the contribution of the singlet PDF to the NS coefficient
-    # and the contribution of the singlet PDF to the S coefficient
-    # do not cancel completely.
+    # compute gluon
     ch_av = tot_ch_sq / len(pids)
-    s_partons = {q: ch_av - ns_partons[q] for q in [*pids, *(-q for q in pids)]}
-    return {"ns": ns_partons, "g": {21: ch_av}, "s": s_partons}
+
+    # NOTE: since here we are implementing the plain coupling
+    # Q1 * Tr[Q2] / nf, we have not th quark sector into singlet
+    # and non singlet
```

Comment:
mh? this sentence doesn't make sense to me ... it seems to miss a verb in the second half ...

Reply:
done in https://github.com/NNPDF/yadism/pull/280/commits/2614f0282e40985eeba9a2676609114786bb1707